docs/docs/adrs/adr-018-permissionless-ics.md

```diff
@@ -0,0 +1,304 @@
+ICS without needing a governance proposal but by simply issuing a transaction.
+
+## Decision
+
+### The Phases of a Consumer Chain
+In Permissionless ICS, launching an Opt In chain is **only** possible through a transaction and not through a `ConsumerAdditionProposal`.
+Nevertheless, Permissionless ICS does not eliminate the `ConsumerAdditionProposal` governance proposal, as proposals are still necessary
+for Top N chains. Because of this, this ADR outlines a solution that attempts to preserve as much of the governance proposal code
+as possible.
+Additionally, to make the distinction between governance-proposed versus transaction-launched chains clearer, in Permissionless ICS,
+we can only add, modify, or remove Top N chains with governance proposals (i.e., `ConsumerAdditionProposal`, `ConsumerModificationProposal`,
+and `ConsumerRemovalProposal`) and we can only add, modify, or remove Opt In chains with transactions.
+In what follows, we describe changes that apply to both governance-proposed (i.e., Top N) and transaction-based (i.e., Opt In) consumer chains.
+
+A consumer chain can reside in three phases: i) _prelaunched_, ii) _launched_, and iii) _stopped_ phase as seen
+in the diagram below:
+![Phases of a consumer chain](./adr18_phases_of_a_consumer_chain.png)
+
+
+When a Top N chain is first proposed through a `ConsumerAdditionProposal` or an Opt In chain is added through a transaction,
+the consumer chain resides in the _prelaunched_ phase. At this state, validators can choose to opt in on the consumer chain. Additionally,
+an Opt In chain can choose to change parameters of the to-be-launched chains, such as `spawnTime`, etc. by issuing a specific transaction.
+This is not the case for Top N chains, where a `ConsumerModificationProposal` can only be issued after a consumer
+chain [has started](https://github.com/cosmos/interchain-security/blob/v4.3.0/x/ccv/provider/keeper/proposal.go#L150).
+
+When the [`spawnTime`](https://github.com/cosmos/interchain-security/blob/v4.3.0/proto/interchain_security/ccv/provider/v1/provider.proto#L55)
+passes and [at least one validator has opted in](https://github.com/cosmos/interchain-security/blob/v4.3.0/x/ccv/provider/keeper/proposal.go#L455)
+the chain can launch and moves to the _launched_ phase. While in launched phase, a Top N consumer chain can choose to modify
+its parameters through a `ConsumerModificationProposal` and an Opt In chain can change its parameters by issuing a transaction.
+
+Lastly, a Top N chain can choose to exit ICS by issuing a `ConsumerRemovalProposal` and an Opt In chain can issue a transaction to stop the chain.
+After some period of time (e.g., provider's unbonding period), all state related to the stopped consumer chain can be removed. We
+keep track of the consumer chain's state for some period, so that we are able to punish validators for misbehaviour.
```

Comment:
```suggestion
keep track of the consumer chain's state for some period, so that we are able to punish validators for misbehaviours that occurred before the consumer chain stopped.
```